projects/sbb-esta/angular-public/src/lib/tooltip/tooltip/tooltip.component.ts

```diff
@@ -146,15 +181,17 @@ export class TooltipComponent implements OnDestroy {
 
   private readonly _closeKeyEventStream = new Subject<void>();
   private _closingActionsSubscription: Subscription;
-
-  constructor(
-    private _overlay: Overlay,
-    private _tooltipRegistry: TooltipRegistryService,
-    @Inject(SBB_TOOLTIP_SCROLL_STRATEGY) private _scrollStrategy,
-    @Optional() @Inject(DOCUMENT) private _document: any,
-    private _zone: NgZone,
-    private _changeDetectorRef: ChangeDetectorRef
-  ) {}
+  /**
+   * Customizations for trigger type - delay open/close
+   */
+  @Input() triggerType: 'click' | 'hover' = 'click';
+  @Input() hoverOpenDelay = 0;
```

Comment:
Refactor hoverOpenDelay and hoverCloseDelay with getter and setter and coerceNumberProperty from the CDK.
Compare with [sbb-textarea maxlength and minlength](https://github.com/SchweizerischeBundesbahnen/sbb-angular/blob/master/projects/sbb-esta/angular-public/src/lib/textarea/textarea/textarea.component.ts#L84).
This would make it more consistent and slightly easier to use.